src/camp_solver.c

```diff
@@ -1891,9 +1827,9 @@ bool is_anything_going_on_here(SolverData *sd, realtype t_initial,
         }
       }
     }
+    printf("DEBUG: is_anything_going_on_here is false, returning success without cvode computing\n");
```

Comment:
should this be wrapped in something like `#ifdef CAMP_DEBUG`?

Reply:
Yeah. Anyway I found a bit awkward that there is no warning about that function. If there is nothing in that cell, it is not a strange behaviour that should be notified to the user? Anyway I will let it inside `#ifdef CAMP_DEBUG` for now.